packages/tailwindcss/src/plugin.cts

```diff
@@ -0,0 +1,5 @@
+// This file exists so that `plugin.ts` can be written one time but compatible with both CJS and ESM
+// without it we get a `.default` export when using `require` in CJS
+
+// @ts-ignore
+module.exports = require('./plugin.ts').default
```

Comment:
There doesn't appear to be a way for tsup to compile a default export to `module.exports` in my testing so we came up with this + two separate entry points in the tsup config. I like this because it feels like it's more clear that it's a workaround. We _might_ be able to do something like this for the postcss plugin too but would need to test.